### PR TITLE
statistics: benchmark selectivity function

### PR DESCRIPTION
**Pull request by tiancaiamao**
`Selectivity` is a critical function that affect the OLTP performance to some degree, 
so add a benchmark function to provide decision when we later optimize it.

### CODE REVIEW

**Comment by tiancaiamao** [comment]
go test -test.bench=BenchmarkSelectivity -test.run=^XXX

```
BenchmarkSelectivity/selectivity-4 100000 21906ns/op  10904B/op  171allocs/op
```

File: statistics/selectivity_test.go [hunk 1]
```diff
@@ -174,3 +190,34 @@ func (s *testSelectivitySuite) TestSelectivity(c *C) {
 		c.Assert(math.Abs(ratio-tt.selectivity) < eps, IsTrue, Commentf("for %s, needed: %v, got: %v", tt.exprs, tt.selectivity, ratio))
 	}
 }
+
+func BenchmarkSelectivity(b *testing.B) {
+	c := &C{}
+	s := &testSelectivitySuite{}
```
Comment (coocood) [inline statistics/selectivity_test.go]: It's a benchmark, we don't need to use the gocheck framework.

**Comment by shenli** [comment]
@lamxTyler PTAL

File: statistics/selectivity_test.go (hunk 1 above)
Comment (tiancaiamao) [reply statistics/selectivity_test.go]: To avoid repeat code here.

Comment (tiancaiamao) [reply statistics/selectivity_test.go]: It's just a struct, if you don't view is as a framework.

**Comment by coocood** [comment]
LGTM

**Pull request merged**